src/tests/ftest/security/pool_security_acl.yaml

```diff
@@ -0,0 +1,66 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+  test_servers:
+    - boro-A
+  test_clients:
+    - boro-A
+  transport_config:
+    - allow_insecure: False
+    - ca_cert: ~/.daos/daosCA.crt
+    - cert: ~/.daos/daos_agent.crt
+    - key: ~/.daos/daos_agent.key
+    - server_name: server
+timeout: 1200
+server_config:
+  name: daos_server
+  port: 10001
+daos_server:
+  start:
+     insecure: False
+dmg:
+  request: pool
+  action: create
+  create:
+    scm_size: 16000000
+pool_acl:
+  user_prefix: daos_ci
+  num_user: 5
+  num_group: 5
+  test_users: !mux
+    user_type1:
+      user:
+         - owner
+    user_type2:
+      user:
+         - user
+    user_type3:
+      user:
+         - ownergroup
+    user_type4:
+      user:
+         - group
+    user_type5:
+      user:
+         - everyone
+  permissions: !mux
+    permission_none:
+      name:
+         - none
+         - deny
+         - deny
```

Comment:
Can you please help me understand why is there "deny" twice under this param name?

Reply:
the first deny is for read, and 2nf deny is for write, this has been docmented.